app/components/phase_banner_component.rb

```diff
@@ -1,19 +1,19 @@
 class PhaseBannerComponent < ViewComponent::Base
   DEFAULT_FEEDBACK_LINK = 'mailto:becomingateacher@digital.education.gov.uk?subject=Feedback%20about%20Apply%20for%20teacher%20training'.freeze
 
-  def initialize(no_border: false, feedback_link: nil)
+  def initialize(no_border: false, feedback_link: DEFAULT_FEEDBACK_LINK)
```

Comment:
This is better than the original code... which I wrote! 😭

Reply:
Don’t worry @duncanjbrown, I won’t tell anyone. Much.